raster/src/main/scala/geotrellis/raster/ArrayMultibandTile.scala

```diff
@@ -107,7 +107,7 @@ class ArrayMultibandTile(_bands: Array[Tile]) extends MultibandTile with MacroMu
     *
     * @return  The object on which the method was invoked
     */
-  def toArrayTile = this
+  def toArrayTile() = this
```

Comment:
Is this required for 2.13? I'm not sure if this an API change for 2.12 or not really.

Reply:
@echeipesh that is not required, and it is not really an API change. The story with this method is that we override here a method that contains `()`: https://github.com/locationtech/geotrellis/blob/2.3/raster/src/main/scala/geotrellis/raster/MultibandTile.scala#L285

Reply:
🤦